keps/sig-cli/20191008-kubectl-events.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Kubectl events
+authors:
+  - "@hpandeycodeit"
+owning-sig: sig-cli
+participating-sigs:
+  - sig-cli
+reviewers:
+  - "@soltysh"
+  - "@pwittrock"
+approvers:
+  - "@soltysh"
+  - "@pwittrock"
+editor: TBD
+creation-date: 2019-10-08
+last-updated: 2019-10-08
+status: provisional
+see-also:
+  - n/a
+replaces:
+  - 
+superseded-by:
+  -
+---
+
+# kubectl events
+
+## Table of Contents
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+    - [Long standing issues](#long-standing-issues)
+  - [Goals](#goals)
+- [Proposal](#proposal)
+  - [Implementation Details/Notes/Constraints [optional]](#implementation-detailsnotesconstraints-optional)
+- [Graduation Criteria](#graduation-criteria)
+<!-- /toc -->
+
+## Summary
+
+Presently, `kubectl get events` has some limitations. It cannot be extended to met the increasing user needs to 
+support more functionality without impacting the `kubectl get`. 
+
+For eg: Any modification to `--watch` functionality for `events` will also change the `--watch` for `kubectl get` since the `events` is dependent of `kubectl get`
+
+Some of the requested features for events include: 
+
+1. Extended behaviour for `--watch` 
+2. Default sorting of `events`
```

Comment:
Can you look through https://github.com/kubernetes/kubernetes/issues and point to all that might be relevant here.
From a quick search I found:
https://github.com/kubernetes/kubernetes/issues/82950
https://github.com/kubernetes/kubernetes/issues/65646 (you've already mentioned above, but worthy a link)
https://github.com/kubernetes/kubernetes/issues/36304
https://github.com/kubernetes/kubernetes/issues/29838 (mentioned above, but I want the link)

Reply:
It would be nice to see if anyone come up with a kubectl plugin that's worthy checking out and syncing with the author.

Reply:
I want to outline all the features requested, not just some.